### PR TITLE
Throatslicing, take II

### PR DESCRIPTION
**Pull request by MarkSuckerberg**
## About The Pull Request
https://github.com/BeeStation/BeeStation-Hornet/projects/2#card-29406288
Port:tgstation/tgstation#46838
Lets you cut someone's throat under certain conditions. Higher damage weapons get the job done faster. Weapon must be sharp, of course.

* Neck grab
* Critical
* Unconsciousness

The bleeding rate is 2.5x Heparin, for comparison.

For me at least, it is a little annoying that there is no way to execute someone in a quick, decisive action with a melee weapon and I am instead stuck stabbing them until they go horizontal. I can go get a gun and do the special execution action, but that's no fun.

## Why It's Good For The Game
It's cool, and it's a much needed way to execute someone a bit faster besides the long and lengthy gun execution.

## Changelog
:cl: Fhaxiris, Mark Suckerberg
add: You cut slice peoples throats now. Select the head on harm intent while they are sleeping, in critical, or neck grabbed.
/:cl:

### CODE REVIEW

**Comment by MarkSuckerberg** [comment]
feck

**Comment by ike709** [comment]
Tested?

File: code/__DEFINES/components.dm [hunk 1]
```diff
@@ -141,6 +141,7 @@
 #define COMSIG_MOB_HUD_CREATED "mob_hud_created"				//! from base of mob/create_mob_hud(): ()
 #define COMSIG_MOB_ATTACK_HAND "mob_attack_hand"				//! from base of
 #define COMSIG_MOB_ITEM_ATTACK "mob_item_attack"				//! from base of /obj/item/attack(): (mob/M, mob/user)
+	#define COMPONENT_ITEM_NO_ATTACK 1
```
Comment (qwertyquerty) [inline code/__DEFINES/components.dm:144]: spaces beforeer

**Review by qwertyquerty — CHANGES_REQUESTED** [review]

File: code/__DEFINES/components.dm (hunk 1 above)
Comment (ike709) [reply code/__DEFINES/components.dm:144]: Intentional.

**Comment by MarkSuckerberg** [comment]
Yes, tested, and it works as intended.

**Pull request merged**